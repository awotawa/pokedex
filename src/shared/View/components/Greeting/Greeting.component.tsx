import React from 'react';
import {GreetingText, GreetingView} from './Greeting.styles';

export const Greeting = () => {
  return (
    <GreetingView>
      <GreetingText>Hello</GreetingText>
    </GreetingView>
  );
};
