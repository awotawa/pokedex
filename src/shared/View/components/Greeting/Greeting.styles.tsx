import styled from '@emotion/native';
import {Text, View} from 'react-native';

export const GreetingText = styled(Text)({
  color: 'green',
  textAlign: 'center',
});

export const GreetingView = styled(View)({
  backgroundColor: 'red',
  width: 70,
  height: 70,
  alignItems: 'center',
  justifyContent: 'center',
  margin: 20,
});
