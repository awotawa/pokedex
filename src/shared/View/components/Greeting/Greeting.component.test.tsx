import * as React from 'react';
import {screen, render} from '@testing-library/react-native';
import {Greeting} from './Greeting.component';

describe('Greeting component', () => {
  it('should render correctly', () => {
    render(<Greeting />);
    expect(screen).toMatchSnapshot();
  });
});
