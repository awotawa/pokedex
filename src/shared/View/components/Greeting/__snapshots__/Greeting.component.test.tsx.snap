// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting component should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "red",
        "height": 70,
        "justifyContent": "center",
        "margin": 20,
        "width": 70,
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "green",
          "textAlign": "center",
        },
        undefined,
      ]
    }
  >
    Hello
  </Text>
</View>
`;
